### PR TITLE
crw-927 Adding 'operatorframework.io/cluster-monitoring: true' label to the OpenShift nightly csv files

### PR DESCRIPTION
**Pull request by ibuziuk**
operatorframework.io/cluster-monitoring: "true" is a pre-requirement for openshift cluster monitoring - https://github.com/openshift/enhancements/blob/master/enhancements/olm/olm-managed-operator-metrics.md#namespace-requirements

Once the label is applied it will be possible to opt in for the cluster monitoring:

image

### CODE REVIEW

**Comment by ibuziuk** [comment]
@tolusha could you please review - would be great to merge it for 7.16.0 ;)

File: olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1594647359/eclipse-che-preview-kubernetes.v9.9.9-nightly.1594647359.clusterserviceversion.yaml.diff [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+@@ -52,13 +52,14 @@
+     categories: Developer Tools
+     certified: "false"
+     containerImage: quay.io/eclipse/che-operator:nightly
+-    createdAt: "2020-07-10T06:21:10Z"
++    createdAt: "2020-07-13T13:35:59Z"
+     description: A Kube-native development solution that delivers portable and collaborative
+       developer workspaces.
+     operatorframework.io/suggested-namespace: eclipse-che
++    operatorframework.io/cluster-monitoring: "true"
```
Comment (davidfestal) [inline olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1594647359/eclipse-che-preview-kubernetes.v9.9.9-nightly.1594647359.clusterserviceversion.yaml.diff:12]: Does it make sense for raw, non-OpenShift Kubernetes installations ?

Comment (ibuziuk) [reply olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1594647359/eclipse-che-preview-kubernetes.v9.9.9-nightly.1594647359.clusterserviceversion.yaml.diff:12]: @davidfestal technically, it should be possible to install on OpenShift via the kubernetes csv, right?
In general, this property would make sense only for CRW, but I thought that CRW csv will fall back on upsteam, no?

Comment (davidfestal) [reply olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1594647359/eclipse-che-preview-kubernetes.v9.9.9-nightly.1594647359.clusterserviceversion.yaml.diff:12]: > it should be possible to install on OpenShift via the kubernetes csv, right?

well, that's not recommended at all. Not even sure it works.

> CRW csv will fall back on upsteam

Not sure what you mean here ? CRW CSV is based on the upstream OpenShift CSV

Are we mixing *upstream* Che and *upstream* Kubernetes here ?

**Comment by ibuziuk** [comment]
@davidfestal PR has been updated adding  'operatorframework.io/cluster-monitoring: true' label only to OpenShift csv

**Comment by ibuziuk** [comment]
[test]

**Comment by flacatus** [comment]
@ibuziuk please ignore minishift tests. I will create a separate Pr to fix them

**Pull request merged**